frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -974,6 +974,9 @@ export const ClusterServiceVersionDetails: React.FC<ClusterServiceVersionDetails
   const csvPlugins = getClusterServiceVersionPlugins(metadata?.annotations);
   const subscription = subscriptionForCSV(props.subscriptions, props.obj);
 
+  let permissions = _.get(spec, 'install.spec.permissions', []);
+  permissions = _.uniqBy(permissions, 'serviceAccountName');
```

Comment:
We typically try to prefer native functions when there is a browser equivalent like optional chaining instead of `_.get` and `Arry.forEach` instead of `_.forEach`. Realistically I don't think we'll be able to remove lodash anytime soon, so I think it's fine for things like `_.uinqBy` which don't have a native equivalent.

Reply:
The main reason to avoid `_.get` is because it loses the TypeScript types. Optional chaining preserves type checking.

Reply:
@spadgett just curious - is there a reason we want to favor native browser equivalent functions over lodash ones? Often times the lodash flavor of functions are more performant. See lodash `foreach` versus native `foreach`: https://www.measurethat.net/Benchmarks/Show/3470/0/lodash-foreach-vs-native-foreach

I have also seen the lodash functions fall through to native functions if the native functions are deemed sufficient.

Reply:
Sorry for the late response. I think I had just started PTO when this was asked.

The main reason to prefer native functions is for better type safety. Definitely for `_.get`. I suspect TypeScript can infer better types for `Array.forEach` compared to `_.forEach`, but I'd need to confirm.

The other issue that comes to mind is what using lodash will mean for dynamic plugins. Do we need to make lodash part of the dynamic plugin API like we're doing for Patternfly? Granted this is a general concern for any dependency and probably a broader discussion we need to have.

We also seem to have multiple versions of lodash (see #2381), which AFAIK we haven't resolved.

If lodash is more performant, I'm open to using it, however.

Adding @christianvogt and @vojtechszocs for other thoughts.